README.md

```diff
@@ -89,7 +89,8 @@ Name               | Description
 -------------------|------------
 debug              | Verbose debug output
 log-format         | Log format, valid options are `txt` (default) and `json`.
-check-keys         | Comma separated list of keys to export value and length/size, eg: `db3=user_count` will export key `user_count` from db `3`. db defaults to `0` if omitted.
+check-keys         | Comma separated list of key patterns to export value and length/size, eg: `db3=user_count` will export key `user_count` from db `3`. db defaults to `0` if omitted. The key patterns specified with this flag will be found using [SCAN](https://redis.io/commands/scan).  Use this option if you need glob pattern matching; `check-keys` is faster for non-pattern keys.
```

Comment:
Last part of this line, instead of `check-keys is faster for non-pattern keys.` - shouldn't it be `check-single-keys is faster for non-pattern keys.` ?

Reply:
Sorry, I just saw this.  You're correct; fixed.